plugins/module_utils/network/ios/argspec/ospfv2/ospfv2.py

```diff
@@ -441,7 +438,22 @@ def __init__(self, **kwargs):
                             },
                             "type": "dict",
                         },
-                        "passive_interface": {"type": "str"},
+                        "passive_interface": {
+                            "options": {
+                                "default": {"type": "bool"},
+                                "interface": {
+                                    "options": {
+                                        "set_interface": {"type": "bool"},
+                                        "name": {
+                                            "type": "list",
```

Comment:
Does the device support a list of interfaces for this command?

Reply:
If so, can we have a test case with the same? I'm not entirely sure about the rendering logic in https://github.com/ansible-collections/cisco.ios/pull/343/files#diff-1d26f6676fd263a0e6c15719081998043a2b6aefbe670ad7baced6750528a9f2R539.

Reply:
yes, the device supports to un-suppress multiple interface at once, ref: https://github.com/ansible-collections/cisco.ios/pull/343/files#diff-2aafdd2b393ea1646b88f3eec30c692711fa078973fe771a3bee662e50a2e896R9, I can include another `interface`